src/envoy/http/authn/authenticator_base.cc

```diff
@@ -68,7 +73,21 @@ bool AuthenticatorBase::validateX509(const iaapi::MutualTls& mtls,
 bool AuthenticatorBase::validateJwt(const iaapi::Jwt& jwt, Payload* payload) {
```

Comment:
You should have new unit tests for this change.

Reply:
Test added.